Cargo.toml

```
@@ -61,6 +61,8 @@ hex = { version = "0.4.3", default-features = false }
 smallvec = "1.11.0"
 serde = { version = "1.0.195", default-features = false, features = ["derive"] }
 serde_json = { version = "1.0.111" }
+serde-big-array = { version = "0.5" }
+bincode = { version = "1.3.3" }
```

This crate has `std` feature ([source](https://github.com/bincode-org/bincode/blob/trunk/Cargo.toml)), so I think we should disable default features and add it to the LP's std feature section?

```suggestion
bincode = { version = "1.3.3", default-features = false }
```

---

Ok, it seems to have it in version 2, not in `1.3.3` which is the latest release in `crates.io`. The version 2 is still marked as RC, last one is: https://github.com/bincode-org/bincode/releases/tag/v2.0.0-rc.3

I see the following options:
- We wait until they make a formal release as `2.0.0`.
- We target to the github branch `2.0.0-rc3`. I understand there are no issues with the library just, the API can change. At least they say it's "usable" 🤔

---

It was not noise! It was a certain comment. We can not keep this merged as 1.3.3 because it will not be compiled for wasm.